forge-gui/res/cardsfolder/upcoming/kyloxs_voltstrider.txt

```diff
@@ -0,0 +1,15 @@
+Name:Kylox's Voltstrider
+ManaCost:1 U R
+Types:Artifact Vehicle
+PT:4/4
+A:AB$ Animate | Cost$ CollectEvidence<6> | Defined$ Self | Types$ Artifact,Creature | SpellDescription$ CARDNAME becomes an artifact creature until end of turn.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPlay | OptionalDecider$ You | TriggerDescription$ Whenever CARDNAME attacks, you may cast an instant or sorcery spell from among cards exiled with it. If that spell would be put into a graveyard, put it on the bottom of its owner's library instead.
+SVar:TrigPlay:DB$ Play | ValidZone$ Exile | Valid$ Instant.ExiledWithSource,Sorcery.ExiledWithSource | ValidSA$ Spell | Controller$ You | Optional$ True | RememberPlayed$ True | Amount$ 1 | SubAbility$ DBEffect
```

Comment:
I feel like the way you wrote those valids you might miss out on something like _Echoing Equation_?

Reply:
Any ideas to fix this ?

Reply:
the Valid$ should be neutral Card since it's only checking the real object, ValidSA is for the spell type check